src/_nachiguro.scss

```diff
@@ -129,77 +129,81 @@ random関数は使用しないでください。
 
 */
 
-@import 'sass-helpers';
+$ncgr-sprockets: null !default;
+$ncgr-assets-prefix: null !default;
+
+@use 'sass-helpers' with (
+  $ncgr-sprockets: $ncgr-sprockets,
+  $ncgr-assets-prefix: $ncgr-assets-prefix
+);
```

Comment:
📝 https://sass-lang.com/documentation/variables/#default-values
>To make this possible, Sass provides the !default flag. This assigns a value to a variable only if that variable isn’t defined or its value is null. Otherwise, the existing value will be used.